Index: backend/src/main/java/G9_LATAM_Team_11_FinanceAI/domain/usuario/Usuario.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package G9_LATAM_Team_11_FinanceAI.domain.usuario;\r\n\r\nimport jakarta.persistence.*;\r\nimport lombok.AllArgsConstructor;\r\nimport lombok.EqualsAndHashCode;\r\nimport lombok.Getter;\r\n\r\nimport java.util.Date;\r\n\r\n\r\n@Table(name=\"usuarios\")\r\n@Entity(name = \"Usuario\")\r\n@Getter\r\n@AllArgsConstructor\r\n@EqualsAndHashCode(of = \"id\")\r\npublic class Usuario {\r\n    @Id\r\n    @GeneratedValue(strategy = GenerationType.IDENTITY)\r\n    private long id;\r\n    private String nombre;\r\n    private String email;\r\n    private String password;\r\n    private double ingreso_mensual;\r\n    private Date fecha_creacion;\r\n    private boolean activo;\r\n\r\n    @OneToMany(mappedBy = \"usuario\")\r\n    private List<Transaccion> transacciones;\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/backend/src/main/java/G9_LATAM_Team_11_FinanceAI/domain/usuario/Usuario.java b/backend/src/main/java/G9_LATAM_Team_11_FinanceAI/domain/usuario/Usuario.java
--- a/backend/src/main/java/G9_LATAM_Team_11_FinanceAI/domain/usuario/Usuario.java	(revision b753147818db52bc7293fcdcd5704242d0c7f760)
+++ b/backend/src/main/java/G9_LATAM_Team_11_FinanceAI/domain/usuario/Usuario.java	(date 1784770126722)
@@ -24,7 +24,4 @@
     private Date fecha_creacion;
     private boolean activo;
 
-    @OneToMany(mappedBy = "usuario")
-    private List<Transaccion> transacciones;
-
 }
Index: backend/src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring.application.name=demo\r\n\r\nspring.datasource.url = jdbc:mysql://${DB_HOST}/FinanceAI?createDatabaseIfNotExist=true\r\nspring.datasource.username=${DB_USER_M}\r\nspring.datasource.password=\r\nspring.datasource.drive-class-name=com.mysql.cj.jdbc.Driver
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/backend/src/main/resources/application.properties b/backend/src/main/resources/application.properties
--- a/backend/src/main/resources/application.properties	(revision b753147818db52bc7293fcdcd5704242d0c7f760)
+++ b/backend/src/main/resources/application.properties	(date 1784806897279)
@@ -2,5 +2,5 @@
 
 spring.datasource.url = jdbc:mysql://${DB_HOST}/FinanceAI?createDatabaseIfNotExist=true
 spring.datasource.username=${DB_USER_M}
-spring.datasource.password=
+spring.datasource.password=${DB_PASS}
 spring.datasource.drive-class-name=com.mysql.cj.jdbc.Driver
\ No newline at end of file
